framework/lib/test_generation/bin/evosuite.sh

```diff
@@ -47,10 +47,12 @@ if [[ $(tail -c1 "$D4J_FILE_TARGET_CLASSES" | wc -l) -eq 0 ]]; then
 fi
 
 # Compute the budget per target class; evenly split the time for search and assertions
-num_classes=$(cat $D4J_FILE_TARGET_CLASSES | wc -l)
+num_classes=$(wc -l "$D4J_FILE_TARGET_CLASSES")
```

Comment:
This looks like the reason for CI failing. The pipeline is intended, because of the following:

> When an option is specified, wc only reports the information requested by that option. 
> The order of output always takes the form of line, word, byte, and file name. 
> ...
>  If no files are specified, the standard input is used and no file name is displayed.

num_classes is processed as an integer. Perhaps, we should refactor this code (here and elsewhere) in a "line_count" function and use a simpler and more robust way of counting new lines.

Reply:
Thanks.  I fixed this by changing it to `wc -l < "$D4J_FILE_TARGET_CLASSES"`.